### PR TITLE
[SDK Renaming] - Fix CFBundleVersion

### PR DESCRIPTION
**Pull request by AlejandroOrozco**
## Jira Ticket
- ACCSEC-

## Github Issue
- ISSUE-

## Description
- Revert CFBundleVersion

## Commit message
-  

## Testing
- [ ] Added unit tests
- [ ] Ran unit tests successfully
- [ ] Added documentation for public APIs and/or Wiki

### CODE REVIEW

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.13%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: TwilioVerifySDK/Info.plist [hunk 1]
```diff
@@ -2,17 +2,25 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>CFBundleDevelopmentRegion</key>
```
Comment (yafuquen) [inline TwilioVerifySDK/Info.plist:5]: Could you keep previous values and only add CFBundleVersion. SDK_* properties were removed

Comment (AlejandroOrozco) [reply TwilioVerifySDK/Info.plist]: Got It, I removed that extra info. Thanks 👍

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.18%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by yafuquen — APPROVED** [review]

**Review by SergioFierro — APPROVED** [review]

**Pull request merged**